guides/doc-Configuring_Load_Balancer/master.adoc

```diff
@@ -8,40 +8,72 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
-//Load Balancing Solution Overview
-include::topics/Overview.adoc[leveloffset=+1]
+include::common/modules/con_load-balancing-solution-architecture.adoc[leveloffset=+1]
 
-// Before You Begin
-include::topics/Before_You_Begin.adoc[leveloffset=+1]
+include::common/modules/con_load-balancing-considerations.adoc[leveloffset=+1]
 
-// Preparing Satellite Server and Capsule Servers for Load Balancing
-include::topics/Preparing_Satellite_Server_and_Capsule_Servers.adoc[leveloffset=+1]
+include::common/modules/con_prerequisites-for-configuring-smart-proxy-server-for-load-balancing.adoc[leveloffset=+1]
 
-// Configuring Capsule Servers for Load Balancing
-include::topics/Configuring_Capsule_Servers_for_Load_Balancing.adoc[leveloffset=+1]
+include::common/modules/con_configuring-smart-proxy-servers-for-load-balancing.adoc[leveloffset=+1]
 
-// Configuring Capsule Server with Default SSL Certificates for Load Balancing without Puppet
-include::topics/Configuring_Capsule_Server_with_Default_SSL_Certificates_for_Load_Balancing_without_Puppet.adoc[leveloffset=+2]
+ifdef::context[:parent-context: {context}]
```

Comment:
I don't 100% understand how all this works; I _assume_ it's like storing the value of "context" in a temporary attribute "parent-context" so that I can switch back after overwriting "context". I copied this from an assembly about Ansible. I use it because I need to include a file twice & it must have unique anchors which contain `{context}`.

Do you think this would work as well:

````diff
-ifdef::context[:parent-context: {context}]
+parent-context: {context}
````

Reply:
Yes, I believe so. Context is always defined in master. Good point about using the same file, I overlooked that.

Reply:
Actually, looking at master again, I don't believe I see the same file twice in there.

Reply:
Yes, because the procedure "proc_creating-custom-ssl-certificates-for-smart-proxy-server.adoc" is part of two workflows.

Reply:
Wouldn't it be better to include it once and refer to it from module located before where it would be included the second time?

Reply:
I believe this is the current default way to handle it. Even though it makes things slightly more complicated/longer, it's IMHO easier to understand/follow for users.

Reply:
Alright. In this case, isn't the change of context needed only for that one module?

Reply:
Technically yes. Would you rather have me only overwrite the context where absolutely necessary? Right now, it kinda "does the same" for all four workflows.

Reply:
Applied.